src/rard/templates/research/antiquarian_detail.html

```diff
@@ -38,244 +38,169 @@
 {% block inner %}
 
     {% with editing as disable_link_controls %}
-    {% with request.user|has_lock:object as has_object_lock %}
+        {% with request.user|has_lock:object as has_object_lock %}
 
 
-    {% if editing %}
-    {% include 'research/partials/disable_main_links.css' %}
-    {% endif %}
+        {% if editing %}
```

Comment:
Presumably `editing` is not going to `== 'introduction'` anymore so line 3 should probably be updated